torch/csrc/jit/ir.h

```diff
@@ -1061,6 +1062,13 @@ friend struct Block;
     return jit::insertConstant(*this, std::move(val), loc);
   }
 
+  // schema-driven insert
+  // this inserts a node into the graph with inputs determined from args and kwargs using Python
+  // argument matching rules, and checks that the op matches a known schema
+  // if this node successfully completes, it guarentees the node is a correctly-formed invocation
+  // of opname
+  Value* insert(Symbol opname, at::ArrayRef<NamedValue> args, at::ArrayRef<NamedValue> kwargs = {});
```

Comment:
It's a bit funny that `args` and `kwargs` have the same type, and `NamedValue` became a bit too overloaded, but I guess that's fine...

Reply:
`NamedValue` is now a poor name. In fact, NamedValue essentially serves the same purpose as `SymbolicVariable` now (i.e. it is a wrapper around Value* that makes it easier to generate code, report errors, and have overloaded operators). 

A future PR can and should just merge NamedValue and SymbolicVariable, choosing a better name.